scripts/kustomize.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#    http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# kustomize.sh: a wrapper for the kubernetes binary used to make it easier to deploy
+#
+# the script will:
+# - download the appropriate binary from github
+# - verify that the checksum is ok
+# - cache the binary for further executions
+
+set -euo pipefail
+
+RELEASE=1.0.6
+
+# See here for the checksums below:
+# https://github.com/kubernetes-sigs/kustomize/releases/download/v1.0.6/checksums.txt
+CHECKSUM_LINUX=8f1eb26f578e5f171a86a13d70bdfbb57249085c5c87ddcbd235b42ff44ab957
+CHECKSUM_DARWIN=fbc35bcf1b67fb81e00334ba0e5508fb421a1bad65d3ef8efc2550aca815964b
+CACHE_DIR=$HOME/.cache/kustomize
+RELEASE_DOWNLOAD_BASE=https://github.com/kubernetes-sigs/kustomize/releases/download/v$RELEASE
+
+# Detect my architecture
+UNAME=$(uname -ps)
+ARCH=""
+case "$UNAME" in 
+	"Darwin i386")
+		ARCH="darwin_amd64"
+		CHECKSUM_EXPECTED="$CHECKSUM_DARWIN"
+		sha256() {
+			shasum -a 256 $1 | awk '{print $1}'
+		}
+		;;
+	"Linux x86_64")
```

Comment:
This is too specific.  My system returns "linux unknown".

Reply:
Yes, I can reproduce this too. I will improve this part.

Reply:
It should be fixed now.